Userland/Libraries/LibWeb/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ set(SOURCES
     CSS/Parser/Declaration.cpp
```

Comment:
There is a typo in the commit title :catdogalert:

Reply:
I stared at the commits for a good 20 seconds...

Reply:
:facepalm: